WordPress/Resources/AppStoreStrings.po

```diff
@@ -40,25 +40,15 @@ msgctxt "app_store_keywords"
 msgid "social,notes,jetpack,writing,geotagging,media,blog,website,blogging,journal"
 msgstr ""
 
-msgctxt "v17.6-whats-new"
+msgctxt "v17.7-whats-new"
 msgid ""
-"With 17.6, you can add the Audio Block to any WordPress.com site, whatever plan you’re on – including the Free Plan. The Audio Block makes it easy to add podcasts, music, and sound files to your posts and pages.\n"
+"If you’ve been wishing someone would send you reminders on the days you want to publish blog posts, your wish has just come true. We’ve added the ability to request blogging reminders. Just choose your preferred blogging days, and we’ll send you a notification, prompting you to post on that day.\n"
```

Comment:
Aside: it's funny that the word by word diff shows a certain pattern "you ... the ... to ... the ..."

The English language really does have certain common structures and frequent words 🤓